### DISCUSSION TITLE
definitions of the new scales?

### DISCUSSION THREAD

**Discussion by maks** [bangcorrupt/hacktribe#134] [Q&A] [answered]
Its awesome to have so many new scales but I must admit for a musical beginner like me its a bit hard to figure out what many of the new scales are from the names. 

For instance I was hoping that one of them would be the [romanian minor (aka ukrainian dorian) scale](https://en.wikipedia.org/wiki/Ukrainian_Dorian_scale) but I'm not sure if its there or even how to figure which one it could be.

Would anyone have any suggestions on how I can figure this out? I'm conscious of not using up @bangcorrupt time on an admin sort of task like this so hoping some one here with more music theory knowledge than I have may have some suggestions?

**Comment by bangcorrupt** [comment, marked as answer]
I think [Lydian Diminished](https://ianring.com/musictheory/scales/1741), it should be in there somewhere.  I use [Ian Ring](https://ianring.com/musictheory/scales/)'s [Amazing Scale Finder](https://ianring.com/musictheory/scales/finder.php) for naming.
